package.json

```diff
@@ -20,9 +20,17 @@
     "test:ci": "cross-env JEST_JUNIT_OUTPUT_DIR=./test-reports JEST_JUNIT_OUTPUT_NAME=unit-results.xml JEST_JUNIT_CLASSNAME=unit yarn run test --ci --reporters=default jest-junit --coverage --coverageDirectory=coverage_report/coverage_maps_unit --coverageReporters=json",
     "test:watch": "jest --watch"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@babel/runtime": "^7.13.10",
+    "css-loader": "^5.2.0",
+    "node-sass": "^5.0.0",
+    "sass-loader": "10.1.1",
```

Comment:
These are not needed. Please sync with main correctly.

Reply:
okay.